src/plugins/dna_export/src/ExportSequenceTask.h

```diff
@@ -70,6 +70,7 @@ class ExportSequenceItem {
 
     void releaseOwnedSeq();
 
+    static bool isRefAlreadyCounted(const U2EntityRef &seqRef);
```

Comment:
If you do not need this method in the public API (aka header file) make it a cpp-local.

Reply:
I'm not agree with this comment.
Because, in this case we can get rid with all private static methods in that way.
But as i understand this is OOP way - to bound methods with entities(classes).
And you suggest to make 'homeless' methods.

Reply:
What is wrong with private static (local) methods? I think they are really good and you can change your mind about them.

A header is a part of the public interface. Public API is a reason why you have the header, otherwise the whole class can be defined in the related CPP file. Adding a private code into a header makes it less readable and adds extra dependencies to users.

If you do not like private statics (aka locals) you can use PImpl pattern (used across QT sources) but in this case you will have to write more boilerplate code: PImpls are good for a statefull code.